package.json

```diff
@@ -1,28 +1,41 @@
 {
-  "name": "notifications-node-client",
-  "version": "4.5.1",
-  "description": "GOV.UK Notify Node.js client ",
-  "main": "index.js",
+  "name": "@govau-platforms/notify-client",
+  "author": "DTA developers",
```

Comment:
`Digital Transformation Agency`

Reply:
`Commonwealth of Australia, DTA` seems to be used in the other js packages. Should we be consistent and go with that?

Reply:
Yep let's go with that for now, thanks for checking.